src/object.d

```diff
@@ -3257,8 +3252,6 @@ private size_t getArrayHash(in TypeInfo element, in void* ptr, in size_t count)
 }
 
 
-// @@@BUG5835@@@ tests:
-
```

Comment:
> If you want to suggest a proper comment, I'd be happy to put it in.

"ensure `TypeInfo_Array.getHash` uses element hash functions instead of hashing array data"



> I don't know. It's not in the scope of this PR.

Can you remove them while you are adding the explanation suggested above? If there is a definition of cruft, this is it -- a comment that has no explanation, and is obviously untrue.